src/components/AppProvider/README.md

```diff
@@ -38,7 +38,21 @@ The app provider component is required to use Polaris. Without it, the component
 AppProvider works by default without any additional options passed to it.
```

Comment:
Most of the changes not in this file are adding `i18n` to app providers and translations to our test provider.